packages/atlas/src/hooks/useWaitForFee.ts

```diff
@@ -0,0 +1,47 @@
+import BN from 'bn.js'
+import { useCallback, useEffect, useState } from 'react'
+
+import { useSubscribeAccountBalance } from '@/providers/joystream/joystream.hooks'
+import { useSnackbar } from '@/providers/snackbars'
+
+export const useWaitForFee = (feeLoading: boolean, fee?: BN, callback?: () => void) => {
```

Comment:
Maybe that's just me and I'm nitpicking here, but the name suggests something different, I would expect that I got fee from this hook. 

Also - I'm not sure if we talk about this during our call, but is it possible to combine this hook with the `useFee` hook? It looks like we're passing `fee` and `feeLoading`, which also feels counter-intuitive. I'm suggesting that we could get a fee from this hook as well. Wdyt?

Reply:
> Also - I'm not sure if we talk about this during our call, but is it possible to combine this hook with the useFee hook? It looks like we're passing fee and feeLoading, which also feels counter-intuitive. I'm suggesting that we could get a fee from this hook as well. Wdyt?

I don't think it's a good idea. `ActionBar` is a good example, now waiting for a fee is handled only there, after your change we would have to handle it in every place where `ActionBar` is used.